src/goto-harness/recursive_initialization.cpp

```diff
@@ -249,6 +258,11 @@ code_blockt recursive_initializationt::build_constructor_body(
     {
       return build_function_pointer_constructor(result_symbol, is_nullable);
     }
+    if(type.subtype().id() == ID_empty)
+    {
+      // always initalize void* pointers as NULL
+      return build_null_pointer(result_symbol);
```

Comment:
This is OK for now, but longer term we'll have to think of a better solution for this (although meaningful void* arguments are tricky, and tbh shouldn't be appearing very often in the sort of harnesses we're trying to generate right now).

Reply:
Feel free to raise a ticket - though tbh I'm not really sure what other useful behaviour exists. A non-det switch over all types? I guess we could do something similar to JBMC where we look at the subsequent casts to pick a type.